src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/CustomerPreferences/GeneralType.php

```diff
@@ -62,10 +64,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Send an email with a summary of the account information after registration.',
                     'Admin.Shopparameters.Help'
                 ),
+                'multistore_configuration_key' => 'PS_CUSTOMER_CREATION_EMAIL',
             ])
-            ->add('password_reset_delay', TextWithUnitType::class, [
+            ->add('password_reset_delay', IntegerType::class, [
                 'label' => $this->trans(
-                    'Password reset delay',
+                    'Password reset delay (in minutes)',
```

Comment:
Why do we specify "in minutes"? It was already written to the right of the field.

![image](https://user-images.githubusercontent.com/70583503/155496655-c022a1ba-aa43-4127-ae8c-c44dfe56cbe1.png)

Reply:
Hi @Julievrz, we had to change field type and this suffix doesn't exist with this new type. Already discussed with @Progi1984 and @okom3pom.

Reply:
Now I'm wondering, maybe we can create a new type `IntegerWithUnitType`? Would make sense IMO.

Reply:
I just searched through all files, `TextWithUnitType` is always used in cases where numerical values (int or float) are expected.

Reply:
We had to change field type because of @marwachelly issue: https://github.com/PrestaShop/PrestaShop/pull/27608#pullrequestreview-884315272

Reply:
Ok then, thanks :-)